lldb/test/API/lang/swift/async/async_fnargs/TestSwiftAsyncFnArgs.py

```diff
@@ -14,6 +14,18 @@ class TestSwiftAsyncFnArgs(lldbtest.TestBase):
     def test(self):
         """Test function arguments in async functions"""
         self.build()
+        self.do_test()
+
+    @swiftTest
+    @skipIf(oslist=['windows', 'linux'])
+    def test_backdeploy(self):
+        """Test function arguments in async functions"""
+        self.build(dictionary={
```

Comment:
Maybe add a comment here that the first macOS to natively support async was macos12?